python/python/tests/test_async_client.py

```diff
@@ -8234,56 +8237,74 @@ async def test_function_stats(self, glide_client: GlideClient):
         )
 
         response = await glide_client.function_stats()
-        check_function_stats_response(response, [], 2, 3)
+        for node_response in response.values():
+            check_function_stats_response(
+                cast(TFunctionStatsSingleNodeResponse, node_response), [], 2, 3
+            )
 
         assert await glide_client.function_flush(FlushMode.SYNC) == OK
 
         response = await glide_client.function_stats()
-        check_function_stats_response(response, [], 0, 0)
+        for node_response in response.values():
+            check_function_stats_response(
+                cast(TFunctionStatsSingleNodeResponse, node_response), [], 0, 0
+            )
 
-    @pytest.mark.parametrize("cluster_mode", [True])
+    @pytest.mark.parametrize("cluster_mode", [False, True])
     @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
-    async def test_function_stats_cluster(self, glide_client: GlideClusterClient):
+    async def test_function_stats_running_script(
```

Comment:
<3
Is it possible to do the same test in node and java?
I can help you with java part if needed.